### PR TITLE
Fix Baozimh.org and rename to GoDa Manhua

### LINKED ISSUE #826: baozimh.org
### Source information

baozimh.org

### Source language

Chinese

### Steps to reproduce

1.Download and install
2.Open
3.Error

### Expected behavior

It works.

### Actual behavior

No result.

### Tachiyomi version

0.16.3

### Android version

Android 9

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.15.3](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### LINKED ISSUE #2141: Chinese source(GoDa漫畫)
### Source name

GoDa漫畫（godamh）

### Source link

**Main**   
https://godamh.com

**Mirror**
https://godamanga.com
https://cocomanga.org

### Source language

Chinese 

### Other details

_No response_

### Acknowledgements

- [X] I have checked that the extension does not already exist by searching the [extension listing](https://keiyoushi.github.io/extensions/) and verified it does not appear there.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a meaningful title with the source name.
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by choppeh**
Closes  #826

Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
- [ ] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

File: src/zh/baozimhorg/src/eu/kanade/tachiyomi/extension/zh/baozimhorg/BaozimhOrg.kt [hunk 1]
```diff
@@ -49,30 +51,18 @@ class BaozimhOrg : HttpSource(), ConfigurableSource {
...
-        val mangas = document.select("article.wp-manga").map { element ->
+        val mangas = document.select(".cardlist .pb-2 a").map { element ->
             SManga.create().apply {
-                val link = element.selectFirst(Evaluator.Tag("h2"))!!.child(0)
-                url = getKey(link.attr("href"))
-                title = link.ownText()
-                thumbnail_url = element.selectFirst(Evaluator.Tag("img"))!!.imgSrc
+                title = element.selectFirst("h3")!!.ownText()
+                thumbnail_url = element.selectFirst("img")?.imgSrc
+                setUrlWithoutDomain(element.absUrl("href"))
```
Comment (AwkwardPeak7) [inline src/zh/baozimhorg/src/eu/kanade/tachiyomi/extension/zh/baozimhorg/BaozimhOrg.kt]: would old urls still work? I belive `getKey` extracts specific part of the url to save to db

Comment (choppeh) [reply src/zh/baozimhorg/src/eu/kanade/tachiyomi/extension/zh/baozimhorg/BaozimhOrg.kt]: Old urls are not reached. Example
Current url:
```sh
https://m.baozimh.one/manga/woduzishengji-duburedicestudio-a04
```
Url from getKey (not working)
```
https://m.baozimh.one/manga/woduzishengji-duburedicestudio
```

**Comment by stevenyomi** [comment]
I'll tweak this in 2 or 3 days.

**Comment by stevenyomi** [comment]
There's an English source (also an aggregator) which I added in the source factory, but it's not included for now because the package is still `zh.baozimhorg`. Chinese users don't use Discord so they don't receive status updates, so it's better not to change the package name for now.

Technically, putting a source factory in `zh` will work correctly now that the source list is flattened. Let me know if you think putting a `zh+en` extension (where Chinese users are major) under `zh.` package prefix is acceptable :)

**Comment by NeKoOuO** [comment]
Why merge?🤔

**Comment by hsiayuchih** [comment]
cool! Finally fixed, been waiting for this for a long time...

**Pull request merged**